README.md

```diff
@@ -13,6 +13,9 @@
 This project is in active and rapid development! Many facets of this project are
 subject to change, and some features are not fully implemented.
 
```

Comment:
May be add a section header ex: `## Certification Documentation` or something like that ?

Reply:
I added a header with the same name that is used on the website.